test/JDBC/input/alter/alter-procedure-vu-verify.sql

```diff
@@ -0,0 +1,66 @@
+exec alter_proc_p1
+go
+
+ALTER PROCEDURE alter_proc_p1
+AS
+    select * from alter_proc_orders
+GO
+
+exec alter_proc_p1
+go
+
+exec alter_proc_p2
+go
+
+ALTER PROCEDURE alter_proc_p1
+    @param INT
+AS
+    IF (@param = 1)
+    BEGIN
+        select * from alter_proc_users
+    END
+
+    ELSE
+    BEGIN
+        select * from alter_proc_orders
+    END
+GO
+
+exec alter_proc_p1 @param = 1
+GO
+
+exec alter_proc_p1 @param = 2
+GO
+
+-- expect error for no param provided
+exec alter_proc_p2
+go
+
```

Comment:
Pls add transaction related case, for example : 
1. begin, do alter, rollback
2. begin, do alter, that alter can't be seen in other session, commit, that alter can be seen in other session
3. begin. do alter. do other row change. commit
4. begin. do alter. do other row change. rollback
5. begin. do alter deliberate result in error. make sure that alter didn't take effect.